symbols/arm7.yml

```diff
@@ -3,55 +3,101 @@ arm7:
     - EU
     - NA
     - JP
+    - EU-WRAM
+    - NA-WRAM
+    - JP-WRAM
+    - EU-RAM
+    - NA-RAM
+    - JP-RAM
   address:
     EU: 0x2380000
     NA: 0x2380000
     JP: 0x2380000
+    EU-WRAM: 0x37F8000
+    NA-WRAM: 0x37F8000
+    JP-WRAM: 0x37F8000
+    EU-RAM: 0x27E0000
+    NA-RAM: 0x27E0000
+    JP-RAM: 0x27E0000
   length:
-    EU: 0x27080
+    EU: 0x27080    #0x1E8 long ENTRY block
     NA: 0x27080
     JP: 0x27080
+    EU-WRAM: 0xF608
+    NA-WRAM: 0xF608
+    JP-WRAM: 0xF608
+    EU-RAM: 0x17878
+    NA-RAM: 0x17878
+    JP-RAM: 0x17878
   description: |-
     The ARM7 binary.
     
     This is the secondary binary that gets loaded when the game is launched.
     
-    Speaking generally, this is the program run by the Nintendo DS's secondary ARM7TDMI CPU, which handles the audio engine, the touch screen, Wi-Fi functions, cryptography, and more.
+    Speaking generally, this is the program run by the Nintendo DS's secondary ARM7TDMI CPU, which handles the audio I/O, the touch screen, Wi-Fi functions, cryptography, and more.
   functions:
     - name: EntryArm7
       address:
         EU: 0x2380000
         NA: 0x2380000
         JP: 0x2380000
       description: |-
-        The entrypoint for the ARM7 CPU. This is like the "main" function for the ARM7 subsystem.
+        The entrypoint for the ARM7 CPU.
         
+        Handles mapping the ARM7 binary into the various memory areas that the program will be using:
+
+        Memory map: (binary is initially loaded at 0x2380000)
+        0x2380000-0x23801E8 => Contains EntryArm7 and two more methods, all related to memory mapping.
+        0x23801E8-0x238F7F0 => Mapped to 0x37F8000, contains MainArm7 and functions crucial to execution.
+        0x238F7F0-0x23A7068 => Mapped to 0x27E0000, contains everything else that won't fit in the fast WRAM.
```

Comment:
Can we move these mapping notes to the top-level binary description?

Reply:
Moved